docs/core/index.md

```diff
@@ -38,10 +41,15 @@ You should see the following output:
 Hello World!
 ```
 
-## Support
+## Participate
 
-.NET Core is [supported by Microsoft](https://dotnet.microsoft.com/platform/support/policy), on Windows, macOS, and Linux. It's updated for security and quality several times a year, typically monthly.
+.NET Core is an open platform. Everyone is welcome to participate.
 
-.NET Core binary distributions are built and tested on Microsoft-maintained servers in Azure and supported just like any Microsoft product.
+* Product issues and questions should be filed as issues at [dotnet/core](https://github.com/dotnet/core/issues)
```

Comment:
We typically send people for product issues to https://developercommunity.visualstudio.com/spaces/61/index.html given that product issues can belong to different repos (runtime, sdk, etc.). Wouldn't that be better?

```suggestion
* Product issues and questions should be filed as issues at [dotnet/core](https://github.com/dotnet/core/issues).
```